otj-server-core/src/main/java/com/opentable/server/EmbeddedJettyBase.java

```diff
@@ -290,7 +290,7 @@ private ConnectorInfo createConnector(Server server, String name,  ServerConnect
         connector.setName(name);
         if (BOOT_CONNECTOR_NAME.equals(name) && bootConnector != null) {
             connector.setHost(bootConnector.getHost());
-            connector.setPort(bootConnector.getPort());
+            connector.setPort(Integer.parseInt(pr.getProperty("server.port", "8080").trim()));
```

Comment:
`bootConnector.getPort()` - always 8080 without SBT customizers. Had to manually push property.